### PR TITLE
Corrected tutorial AssetManager

### PR DESCRIPTION
**Pull request by dad72**
I add a demo to the playground and I corrected some small errors

### CODE REVIEW

File: content/tutorials/03_Advanced/How_to_use_AssetsManager.md [hunk 1]
```diff
@@ -66,32 +66,55 @@ The manager itself provides three callbacks:
...
+* Playground demo [here](http://www.babylonjs-playground.com/#ZJYNY#0)
 
 ## Using loading screen
 
 By default, the AssetsManager will display a loading screen while loading assets:
 
 ![](http://www.babylonjs.com/Screenshots/assets.jpg)
 
-If you want to disable the loading screen, you have to set ```assetsManager.useDefaultLoadingScreen``` to false.
+If you want to disable the loading screen, you have to set to false:
```
Comment (deltakosh) [inline content/tutorials/03_Advanced/How_to_use_AssetsManager.md]: Why removing the property name in this sentence?

File: content/tutorials/03_Advanced/How_to_use_AssetsManager.md [hunk 2]
```diff
@@ -66,32 +66,55 @@ The manager itself provides three callbacks:
...
+```
+
+Loading text is controlled using
+
+```javascript
+engine.loadingUIText = "text";
+```
 
-Loading text is controlled using ```engine.loadingUIText = "text"```
+Background color is controlled using ```loadingUIBackgroundColor``` for instance.
```
Comment (deltakosh) [inline content/tutorials/03_Advanced/How_to_use_AssetsManager.md]: Why 'for instance'

File: content/tutorials/03_Advanced/How_to_use_AssetsManager.md (hunk 1 above)
Comment (dad72) [reply content/tutorials/03_Advanced/How_to_use_AssetsManager.md]: By this I rewrote the following full if you look at all the file.

File: content/tutorials/03_Advanced/How_to_use_AssetsManager.md (hunk 2 above)
Comment (dad72) [reply content/tutorials/03_Advanced/How_to_use_AssetsManager.md]: Not necessarily useful, but it is to give the sample code below.

Comment (dad72) [reply content/tutorials/03_Advanced/How_to_use_AssetsManager.md]: Je crois qu'il y a erreur de traduction. je voulais ecrire "par exemple : "

File: content/tutorials/03_Advanced/How_to_use_AssetsManager.md (hunk 1 above)
Comment (deltakosh) [reply content/tutorials/03_Advanced/How_to_use_AssetsManager.md]: Yep but the sentence must be "you have to set assetsManager.useDefaultLoadingScreen to false"

**Comment by dad72** [comment]
I fix what was bad.

File: content/tutorials/03_Advanced/How_to_use_AssetsManager.md (hunk 2 above)
Comment (deltakosh) [reply content/tutorials/03_Advanced/How_to_use_AssetsManager.md]: Just remove 'for instance':)

**Comment by deltakosh** [comment]
Perfect thanks a lot

**Comment by dad72** [comment]
Au faite, le "for instance" était déjà écrit dans la doc actuel (tout en bas, dernière ligne) je ne l'avais pas écrit:
http://doc.babylonjs.com/tutorials/How_to_use_AssetsManager

**Comment by deltakosh** [comment]
Ok:)

**Pull request merged**